round.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * rounds a value to a certain number of decimals
+ *
+ * @exports round
+ * @kind function
+ *
+ * @example
+ * import round from '@datawrapper/shared/round';
+ * round(1.2345); // 1
+ * round(1.2345, 2); // 1.23
+ * round(12345, -2); // 12300
+ *
+ * @param {number} value - the value to be rounded
+ * @param {number} decimals - the number of decimals
+ * @returns {string} - numeral.js compatible format string
+ */
+export default function round(value, decimals = 0) {
+    const base = Math.pow(10, decimals);
+    return Math.round(value * base) / base;
```

Comment:
This returns a number but in the JSDoc comment you say it should return a numeral compatible `string`. Which one is correct?

Reply:
fixed the jsdoc